autosuggest-helper-tutorial.md

```diff
@@ -5,17 +5,12 @@
 
 This tutorial is intended for anyone that already has an autocomplete UI element in their app showing possible addresses in a AutoCompleteTextView/EditText/RecyclerView. It explains a method of adding what3words suggestions alongside your existing address results.
 
-## Example
-
-There is a sample provided called [sample-multi-autosuggest-providers](https://github.com/what3words/w3w-android-wrapper/tree/master/sample-multi-autosuggest-providers) on our GitHub repo folder of the API wrapper repository. 
-The example uses a EditText and a RecyclerView with two different providers for location autosuggest, **Google Places API** and **what3words API** using our AutosuggestHelper to show the 3 words addresses in the manner described below.  
-
```

Comment:
I could not find this sample app anywhere, maybe it was deleted permanently.

Reply:
Ok cool probably we should do something using jetpack compose anyway, I will add this to the backlog for now